frontend/public/index.html

```diff
@@ -25,6 +25,8 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>React App</title>
+
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/instantsearch.css@7/themes/algolia-min.css">
```

Comment:
In https://www.algolia.com/doc/guides/building-search-ui/widgets/customize-an-existing-widget/react/#loading-the-theme section there are several methods described. Why have you chosen CDN way instead of npm & Webpack? Just curious.

Reply:
```import 'instantsearch.css/themes/algolia.css';``` is now in ```App.js``` and works as expected. Removed ```CDN``` link in ```index.html```